src/createClient.js

```diff
@@ -0,0 +1,45 @@
+const { Client } = require('./client')
+const assert = require('assert')
+
+module.exports = { createClient }
+
+/** @param {{ version?: number, hostname: string, port?: number }} options */
+function createClient (options) {
+  assert(options && options.hostname)
+  const client = new Client({ port: 19132, ...options })
```

Comment:
Add port to options

Reply:
it uses rest operator, so iirc it'll just overwrite the port from options if its set

Reply:
Ok